apps/experiments/serializers.py

```diff
@@ -1,28 +1,63 @@
 from uuid import UUID
 
-from experiments.models import Experiment
-from pydantic import BaseModel
+from experiments.models import Experiment, SourceMaterial
+from pydantic import BaseModel, Field, HttpUrl
+
+
+class SafetyLayerData(BaseModel):
+    name: str
+    prompt_text: str
+    messages_to_review: str
+    default_response_to_user: str | None = ""
+    prompt_to_bot: str | None = ""
+
+
+class SurveyData(BaseModel):
+    name: str
+    url: HttpUrl
+    confirmation_text: str
+
+
+class ConsentFormData(BaseModel):
+    name: str
+    consent_text: str
+    capture_identifier: bool
+    identifier_label: str
+    identifier_type: str
+    is_default: bool
+    confirmation_text: str
+
+
+class SyntheticVoiceData(BaseModel):
+    name: str
+    neural: bool
+    language: str
+    language_code: str
+    gender: str | None
+    service: str
+    voice_provider: str | None
+    file: str | None
```

Comment:
I'm not sure what to do with this but I do think we should keep a FK to the file to make sure we can track references.

One option is to have a field `ExperimentVersion.voice_file`.

Another is to group all the referenced files together in `ExperimentVersion.files` and keep the ID reference in the JSON data as well.